WorkflowAdminBundle/EventSubscriber/AddGroupWorkflowProfileSubscriber.php

```diff
@@ -89,6 +94,28 @@ public function addWorkflowProfile(GroupFormEvent $event)
             'sub_group_id' => 'backoffice',
         ));
         $builder->get('workflow_profile_collections')->addModelTransformer($this->workflowProfileCollectionTransformer);
+        $builder->addEventSubscriber($this);
+
+    }
+
+    /**
+     * @param FormEvent $event
+     */
+    public function postSubmit(FormEvent $event)
+    {
+        if (($group = $event->getData()) instanceof GroupInterface) {
+            $workflowProfileCollections = $group->getWorkflowProfileCollections();
+            $perimeter = new Perimeter();
+            $perimeter->setType(ContentTypeInterface::ENTITY_TYPE);
+
+            foreach ($workflowProfileCollections as $key => $workflowProfile) {
+                if ($key != NodeInterface::ENTITY_TYPE && count($workflowProfile->getProfiles()) > 0) {
+                    $perimeter->addItem($key);
+                }
+            }
+
```

Comment:
@itkg-nanne : I'm not sure at all about this perimeter ? What is saved here ? Is it for the following use case : Contributor on News and Validator on Cars ?